data_explorer/data_explorer.py

```diff
@@ -0,0 +1,59 @@
+import gradio as gr
+
+from loader import load_data
+
+data = load_data()
+
+with gr.Blocks() as demo:
+    assistant_roles = data['assistant_roles']
+    user_roles = data['user_roles']
+    with gr.Row():
```

Comment:
I think we miss a dropdown for tasks. The `(user, assistant)` pair will have multiple tasks to perform.